site/learn/expert-advice/oauth/whats-new-with-oauth-2-1.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: advice
+title: What is new in the OAuth 2.1?
+description: A draft of the OAuth 2.1 specification was recently released. What's coming down the pike?
+image: advice/oauth-device-authorization-article.png
+author: Dan Moore
+category: OAuth
+date: 2020-10-20
+dateModified: 2020-10-20
+---
+
+OAuth is getting spiffed up a bit. The original OAuth 2.0 specification was released in October 2012 as [RFC 6749](https://tools.ietf.org/html/rfc6749) and [RFC 6750](https://tools.ietf.org/html/rfc6750). It replaced OAuth 1.0, released in April 2010. There have been a number of extensions and modifications to OAuth 2 over the subsequent years. A new OAuth specification has been proposed and is currently under discussion. At this time, the specification was most recently updated on July 30, 2020. If approved, [OAuth 2.1](https://tools.ietf.org/html/draft-ietf-oauth-v2-1-00) will obsolete certain parts of Oauth 2.0 and mandate security best practices. The rest of the OAuth 2.0 specification will be retained. 
+
+That bears repeating. Nothing new will be added. This is an explicit design goal of OAuth 2.1.
+
+This article assumes you are a developer or have analogous technical experience and are familiar with OAuth and the terms used in the various corresponding standards. If you’d like an introduction to OAuth and why you’d consider using it, [Wikipedia is a good place to start](https://en.wikipedia.org/wiki/OAuth). This article covers proposed changes to OAuth that might affect you if you are either using OAuth in your application or if you build software which implements OAuth. 
```

Comment:
Maybe link to https://fusionauth.io/learn/expert-advice/oauth/oauth-v1-signed-requests/? Maybe: "If you’d like an introduction to OAuth and why you’d consider using it, check out this Wikipedia overview and our expert advice article on V1 signed requests"

Reply:
Yes, doesn't really fit, unfortunately. When Brian releases the OAuth guide, we should point to that.